homeassistant/components/oru/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+    "domain": "oru",
+    "name": "Orange and Rockland Utility Smart Energy Meter Sensor",
+    "documentation": "https://www.home-assistant.io/components/",
```

Comment:
Please link to the correct docs page of the integration.

Reply:
Done